### ISSUE TITLE
CoinEx fetch_balance method not working

### ISSUE THREAD

**Issue by mkang1234** [ccxt/ccxt#5956]
- OS: Windows 10
- Programming Language version: Python
- CCXT version: 1.18.1115
- Exchange: CoinEx
- Method: fetch_balance

The function doesn't return any balance. Seems to fail in the following line of code in function fetch_balance. 
response = self.privateGetBalanceInfo(params)

**Comment by kroitor** [comment]
@mkang1234 can you plz post your verbose request/response ?

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue
- https://github.com/ccxt/ccxt/wiki/FAQ#i-am-calling-a-method-and-i-get-an-error-what-am-i-doing-wrong

**Comment by kroitor** [comment]
```Python
exchange.load_markets()
exchange.verbose = True
exchange.fetch_balance()
```

↑ We need your verbose request/response to help.

Also, does it happen with the most recent version of CCXT?

**Comment by mkang1234** [comment]
Here you go. I tried latest using version 1.18.1242 with same result. 

Request: GET https://api.coinex.com/v1/balance/info?access_id=XXXXXXXXXXXXXXXXXXXXX {'Authorization': 'XXXXXXXXXXXXXXXXXX, 'Content-Type': 'application/json', 'User-Agent': 'python-requests/2.20.0', 'Accept-Encoding': 'gzip, deflate'} None

Response: GET https://api.coinex.com/v1/balance/info?access_id=XXXXXXXXXXXXXXXXXXX=1570775510631 200 {'Date': 'Fri, 11 Oct 2019 06:31:52 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=d9f0fdd190fd3f73e15d0332e9c9066b81570775512; expires=Sat, 10-Oct-20 06:31:52 GMT; path=/; domain=.coinex.com; HttpOnly; Secure', 'Vary': 'Accept-Encoding', 'Content-Encoding': 'gzip', 'CF-Cache-Status': 'DYNAMIC', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '523ecb263e63d362-LAX'} {"code":0,"data":{},"message":"Ok"}

**Comment by kroitor** [comment]
@mkang1234 it looks like you have no balance or zero balance with coinex, right?

**Comment by mkang1234** [comment]
Correct I have no balance.

**Comment by kroitor** [comment]
@mkang1234 and in the Manual it is documented that:

> Some exchanges may not return full balance info. Many exchanges do not return balances for your empty or unused accounts. In that case some currencies may be missing in returned balance structure.

https://github.com/ccxt/ccxt/wiki/Manual#balance-structure

Right? )

**Comment by mkang1234** [comment]
I was expecting the function to just return an empty list or dictionary structure when calling the function if there is no balance. What happens is that the function call hits the exception part of the code.  

        try:
            self.api.verbose = True
            balance_data_raw = self.api.fetch_balance()
        except:
            Gets to this part when calling the function.

**Comment by kroitor** [comment]
@mkang1234 ok, can you show the exception itself or post an error traceback?

**Comment by kroitor** [comment]
@mkang1234 

```Python
    try:
        self.api.verbose = True
        balance_data_raw = self.api.fetch_balance()
    except Exception as e:
        print(type(e).__name__, str(e))
```

**Comment by mkang1234** [comment]
print(e) ---->  Ok
print(e, type(e).__name__, str(e)) ---->   ExchangeError Ok


Traceback (most recent call last):
  File "C:\Users\mkang\OneDrive\OmniBlock\trader_app\CoinEx_Class.py", line 757, in get_all_balances
    balance_data_raw = self.api.fetch_balance()
  File "C:\Users\mkang\Anaconda3\lib\site-packages\ccxt\coinex.py", line 360, in fetch_balance
    response = self.privateGetBalanceInfo(params)
  File "C:\Users\mkang\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 436, in inner
    return entry(_self, **inner_kwargs)
  File "C:\Users\mkang\Anaconda3\lib\site-packages\ccxt\coinex.py", line 851, in request
    raise ErrorClass(response['message'])
ccxt.base.errors.ExchangeError: Ok

**Comment by kroitor** [comment]
@mkang1234 ok, thx! Apparently, this was a bug in the empty response handling, so i've uploaded a fix for it, and checked on my side – should work fine now. This is fixed in CCXT version 1.18.1243+ and it will be available for downloading in 15 minutes. We will be happy if you let us know whether it does resolve the issue for you or not. Thx for reporting!

**Comment by mkang1234** [comment]
Looks like it's fixed now. Thank you for your help and quick response!!
